tox.ini

```diff
@@ -1,42 +1,50 @@
 [tox]
-envlist = py36-flake8, py36-pylint, py36-test, py36-bandit
+envlist = py{36,37-dev,py3},flake8,pylint,test,bandit
 skipsdist=True
+skip_missing_interpreters = {env:TOX_SKIP_MISSING_INTERPRETERS:True}
+app = nim
```

Comment:
Creating a variable so we don't have to hardcode it multiple times. If someone finds a way to get that app name dynamically (i.e. without having to manually define it here), I'll give you 1000 Internet points.